layout/_macro/sidebar.swig

```diff
@@ -19,10 +19,10 @@
       {%- endif %}
 
       <ul class="sidebar-nav motion-element">
-        <li class="sidebar-nav-toc" data-target="post-toc-wrap">
+        <li class="sidebar-nav-toc" data-target="0">
```

Comment:
😰 I don't think there should be dynamic numbers like 0 1 2...

Reply:
You could find a better workaround~